Description: Do not run cli.js as it depends on many packages.
 Get the version from package.json instead.
Forwarded: https://github.com/isaacs/npm/pull/3873
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2013-09-08
Index: npm/scripts/doc-build.sh
===================================================================
--- npm.orig/scripts/doc-build.sh
+++ npm/scripts/doc-build.sh
@@ -4,7 +4,7 @@ src=$1
 dest=$2
 name=$(basename ${src%.*})
 date=$(date -u +'%Y-%m-%d %H:%M:%S')
-version=$(node cli.js -v)
+version=$(nodejs -e "console.log(require('./package').version)")
 
 mkdir -p $(dirname $dest)
 
